woocommerce/woocommerce
# adding states for Benin country 🇧🇯

I've added some states for the Benin country 

### All Submissions:






### Changes proposed in this Pull Request:



These changes add all states for Benin country to the list of WooCommerce Countries. It give the ability to the customers,  to pick and select the right states while he is buying on a WooCommerce store.
Closes #

### How to test the changes in this Pull Request:

1. Pick a product on your store
2. On the checkout page select Benin as country 
3. If you have selected Benin as country, you will need to pick the state name you want to pick and then proceed the order.
4. Go to the WooCommerce order for a Benin customer and you'll see the state name displayed into the order

### Changelog entry

> Adding states for Benin country 🇧🇯
